.github/workflows/rust.yml

```
@@ -147,6 +147,7 @@ jobs:
           - mipsel-unknown-linux-gnu
           - powerpc-unknown-linux-gnu
           - powerpc64-unknown-linux-gnu
+          - wasm32-unknown-unknown
```

```suggestion
          - wasm32-unknown-unknown
          - wasm32-unknown-emscripten,
          - wasm32-wasi
```

Would it make sense to test against these targets as well?

---

I think `wasm32-unknown-unknown` is the most restrictive one. If it works for those, it should work for the others too :)

---

Understood. Let's proceed as is then.